src/_cffi_src/openssl/bio.py

```diff
@@ -11,37 +11,8 @@
 TYPES = """
 typedef struct bio_st BIO;
```

Comment:
`CRYPTO_mem_leaks(struct bio_st *);` is (obnoxiously) a function signature. We can do this if we want to move this function to MACROS and change its definition though.